tgmc.dme

```diff
@@ -116,7 +116,6 @@
 #include "code\_onclick\hud\robot.dm"
 #include "code\_onclick\hud\screen_objects.dm"
 #include "code\controllers\_DynamicAreaLighting_TG.dm"
-#include "code\controllers\authority.dm"

```

Comment:
undocumented change?

Reply:
Not really it is a 3 line file that's useless now since I'm getting rid of the final authority